tests/pathogen-embed-auto.t

```diff
@@ -0,0 +1,12 @@
+Run pathogen-embed with t-SNE on a H3N2 HA alignment.
```

Comment:
Can you update this first line of this test and the other test in this PR to say what the test is doing more specifically? We just need a way to know that this test is checking whether `--learning-rate auto` works compared to the existing t-SNE tests which specify a learning rate value.

Reply:
Just pushed changes to fix this!